quick-start-headless-react-custom/src/card-objects/TextBlock.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2022 Workgrid Software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react'
+import * as AC from 'adaptivecards'
+import { reactDomRender } from './shared'
+import Typography from '@mui/material/Typography'
+
+const fontMappings = [
```

Comment:
The purpose here is to map the adaptive card element's size property to the appropriate material UI typography variant (h1,h2,body1, etc). Otherwise, the Typography wrapper would not size the text appropriately. Comments make sense here, so I'll go ahead and add one.